Doc/license.rst

```diff
@@ -788,7 +788,7 @@ of the OpenSSL license here::
      * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
      * SUCH DAMAGE.
      *
-     * The licence and distribution terms for any publically available version or
+     * The licence and distribution terms for any publicly available version or
```

Comment:
You can't just change the license text, especially if this is a license of imported code.